snuba/cli/bootstrap.py

```diff
@@ -106,11 +108,28 @@ def bootstrap(
     # tables or distributed tables, etc.
 
     # Create the tables for every dataset.
+    existing_tables = {row[0] for row in conn.execute("show tables")}
     for name in DATASET_NAMES:
         dataset = get_dataset(name)
 
         logger.debug("Creating tables for dataset %s", name)
+        run_migrations = False
         for statement in dataset.get_dataset_schemas().get_create_statements():
-            logger.debug("Executing:\n%s", statement)
-            ClickhousePool().execute(statement)
+            if statement.table_name not in existing_tables:
+                # This is a hack to deal with Materialized views.
+                # All our DDL create statements are in the for of CREATE IF NOT EXISTS
+                # This works well for all tables, except for Materialized views.
+                # It seems Clockhouse parses the SQL query that defines the view even if
```

Comment:
```suggestion
                # It seems ClickHouse parses the SQL query that defines the view even if
```

I do like Clockhouse though. 😄